leftwm-core/src/utils/command_pipe.rs

```diff
@@ -58,7 +58,7 @@ impl CommandPipe {
     pub fn pipe_name() -> PathBuf {
         let display = env::var("DISPLAY")
             .ok()
-            .and_then(|d| d.rsplit_once(":").map(|(_, r)| r.to_owned()))
+            .and_then(|d| d.rsplit_once(':').map(|(_, r)| r.to_owned()))
```

Comment:
Is it of any meaning to use single quotes here? Just asking for educational purpose..

Reply:
Was actually a clippy, but the reason is single quotes represents a char whereas double represents a string. Idk hope the backend works exactly but I would presume in the compiler the string would be optimised into a char, as type char has a smaller sized memory slot.